cli.js

```diff
@@ -1,42 +1,37 @@
 #!/usr/bin/env node
+'use strict';
 
-"use strict";
+require('v8-compile-cache');
+const logger = require('./lib/utils/logger');
 
-require("v8-compile-cache");
-
-const importLocal = require("import-local");
+const importLocal = require('import-local');
 
 // Prefer the local installation of webpack-cli
 if (importLocal(__filename)) {
-	return;
+    // @ts-ignore
```

Comment:
The compiler gives error because a `return` is not allowed outside a function. For now it's ignored but shouldn't we use `process.exit`?

Reply:
Depends on what we want to do, import local is ensuring that we're not in a global - local scope or vice versa

https://github.com/sindresorhus/import-local#readme